### ISSUE TITLE
Colab Benchmark

### ISSUE THREAD

**Issue by JoshuaC3** [mikecroucher/Jupyter-Matrix-Matrix#1]
Here is a benchmark for Colab, using your code from the other notebooks.

![image](https://user-images.githubusercontent.com/11645712/32729763-c193fa72-c87c-11e7-986a-410d4282db72.png)

```
peak_flops[0]

47.66705396123174
```

took a long time to run but didn't timeout.

You might be able to access through here https://colab.research.google.com/notebook#fileId=1C22e2fjyhzzxV2Uo4sOx2_0U3iqbetmM&scrollTo=vqA0l1WBtnCp

**Comment by mikecroucher** [comment]
Thanks. That's a strange looking curve!

**Comment by tonywangcn** [comment]
Thanks for your benchmark, have you tested it with aws and paperspace?

**Comment by JoshuaC3** [comment]
No, sorry. Would be good to see more of these. I'm  not sure that paperspace is though!
